new-toast-site/src/page-wrapper.js

```diff
@@ -11,12 +11,12 @@ import { Flex } from "theme-ui";
 import Header from "./temp/Header.js";
 import Copyright from "./temp/Copyright.js";
 
-const StyledSummary = styled.summary`
+const StyledSummary = styled.default.summary`
```

Comment:
it would seem theme-ui's exports result in a styled.default when imported. Not sure what's going on here but re: the PR description we may need to do something to theme-ui anyway